cibyl/utils/files.py

```diff
@@ -13,11 +13,18 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 """
-import logging
 import os
 from os import PathLike
 
-LOG = logging.getLogger(__name__)
+from cibyl.cli.parser import LOG
```

Comment:
I would not use the parser LOG here, it could be confusing.

Reply:
> I think that we should have some mechanism to turn off color output, in case for example the user wants to redirect the output to a file. I am not sure how we could do that, maybe having method in the Colors class that would set all the attributes to an empty string?

That is actually a great idea.
I will explore it.
Thank you!